### PR TITLE
sbt-typelevel-ci-release

### PR DESCRIPTION
**Pull request by tpolecat**

### CODE REVIEW

**Comment by armanbilge** [comment]
You should remove the `scalaVersion` setting, and set `ThisBuild / crossScalaVersions`. If `scala213` is last in the list, it will be your default `scalaVersion` when you start sbt. (This is all inherited from sbt-gh-actions btw.)

```
    scalaVersion := scala213Version,
    crossScalaVersions := Seq(scala212Version, scala213Version, scala30Version),
```

File: build.sbt [hunk 1]
```diff
@@ -23,31 +23,46 @@ lazy val scala213Version      = "2.13.7"
...
-// These are releases to ignore during MiMa checks
-lazy val botchedReleases = Set("0.8.0", "0.8.1")
+// Basic versioning and publishing stuff
+ThisBuild / tlBaseVersion := "1.0"
+ThisBuild / scalaVersion := scala213Version
+ThisBuild / crossScalaVersions := Seq(scala212Version, scala213Version, scala30Version)
+ThisBuild / developers += tlGitHubDev("tpolecat", "Rob Norris")
+ThisBuild / tlSonatypeUseLegacyHost := false
+ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"))
+ThisBuild / githubWorkflowBuildPreamble ++= Seq(
```
Comment (armanbilge) [inline build.sbt:33]: FYI there's also `githubWorkflowBuild` (the default `ci` step resides in here) and `githubWorkflowBuildPostamble`.

Comment (tpolecat) [reply build.sbt:33]: Hm yeah maybe I should do docs last.

**Comment by tpolecat** [comment]
Alright let's see how it does.

**Pull request merged**